interactive/management/commands/resubmit.py

```diff
@@ -0,0 +1,16 @@
+from django.core.management.base import BaseCommand
+
+from interactive.models import AnalysisRequest
+from interactive.submit import submit_analysis
+
+
+class Command(BaseCommand):
+    help = "Resubmit an analysis request (commit, push, and resubmit job)"  # noqa: A003
+
+    def add_arguments(self, parser):
+        parser.add_argument("analysis_request_ids", nargs="+")
+
+    def handle(self, *args, **options):
+        for ar_id in options["analysis_request_ids"]:
+            r = AnalysisRequest.objects.get(pk=ar_id)
```

Comment:
If this doesn't exist could we tell the user without a stacktrace?

Reply:
The use being devs, I thought that would be fine?

Reply:
Devs like clear feedback too 😢

Reply:
What is clearer than a traceback? :rofl:

Reply:
So, this is not a 'cli app' IMO

It is a one off tool we use occasionally.

So I don't think it warrants the same level of effort.

That said, the timeflake error is confusing, so I will try to handle it.

however, there's some time pressure as submitted jobs have failed

Reply:
Updated to handle both invalid timeflake ids and missing db entries more gracefully

Reply:
Well it's running on the CLI so IMO it's an "app" 😜

IME these little tools tend to hang around so I like to add a tiny bit of known error handling to just help myself out!  That being said you've done more than enough here 👌